plugin/src/main/java/com/gtnewhorizons/retrofuturagradle/modutils/ModUtils.java

```diff
@@ -105,6 +105,16 @@ public ModUtils(Project project, MinecraftExtension mcExt, MinecraftTasks minecr
                     "Apply MCP decompiler cleanup to the main source set, doing things like replacing numerical OpenGL constants with their names");
         });
 
+        project.getTasks().register("migrateMappings", MigrateMappingsTask.class, task -> {
+            task.setGroup(TASK_GROUP_USER);
+            task.setDescription(
+                    "Migrate main source set to a new set of mappings");
+            task.dependsOn(project.getTasks().getByName("extractMcpData"));
```

Comment:
Use getTasks().named("...") instead of getByName, named is lazy while getByName will fully construct and configure the task.